test/spec/fixtures/privileges.sql

```diff
@@ -1,221 +1,34 @@
-    , descendant
-    , being_part
-    , part
-    , leak
-    , perf_articles
-    , employees
-    , tiobe_pls
-    , single_unique
-    , compound_unique
-    , only_pk
-    , family_tree
-    , managers
-    , organizations
-    , authors
-    , publishers
-    , books
-    , forties_books
-    , fifties_books
-    , sixties_books
-    , person
-    , message
-    , person_detail
-    , space
-    , zone
-    , projects_dump
-    , "UnitTest"
-    , json_arr
-    , jsonb_test
-    , authors_books_number
-    , authors_have_book_in_decade
-    , authors_have_book_in_decade2
-    , forties_and_fifties_books
-    , odd_years_publications
-    , foos
-    , bars
-    , materialized_projects
-    , contract
-    , player_view
-    , contract_view
-    , ltree_sample
-    , isn_sample
-    , projects_count_grouped_by
-    , "Server Today"
-    , pgrst_reserved_chars
-    , authors_w_entities
-    , openapi_types
-    , openapi_defaults
-    , getallprojects_view
-    , get_projects_above_view
-    , web_content
-    , pages
-    , referrals
-    , big_projects
-    , sites
-    , jobs
-    , main_jobs
-    , whatev_projects
-    , whatev_sites
-    , whatev_jobs
-    , agents
-    , departments
-    , schedules
-    , activities
-    , unit_workdays
-    , unit_workdays_fst_shift
-    , stuff
-    , loc_test
-    , v1.parents
-    , v2.parents
-    , v2.another_table
-    , v1.children
-    , v2.children
-    , screens
-    , labels
-    , label_screen
-    , actors
-    , films
-    , personnages
-    , end_1
-    , end_2
-    , schauspieler
-    , filme
-    , rollen
-    , products
-    , suppliers
-    , products_suppliers
-    , trade_unions
-    , suppliers_trade_unions
-    , client
-    , clientinfo
-    , contact
-    , chores
-    , limited_update_items
-    , limited_update_items_cpk
-    , limited_update_items_no_pk
-    , limited_update_items_view
-    , limited_update_items_wnonuniq_view
-    , limited_delete_items
-    , limited_delete_items_cpk
-    , limited_delete_items_no_pk
-    , limited_delete_items_view
-    , plate
-    , well
-    , limited_delete_items_wnonuniq_view
-    , limited_delete_items_cpk_view
-    , limited_update_items_cpk_view
-    , xmltest
-    , oid_test
-    , job
-    , series
-    , adaptation_notifications
-    , series_popularity
+GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA
+      "SPECIAL ""@/\#~_-"
+    , "تست"
     , test
-    , view_test
-    , shops
-    , shop_bles
-    , "SPECIAL ""@/\#~_-".languages
-    , "SPECIAL ""@/\#~_-".names
-    , do$llar$s
-    , safe_update_items
-    , safe_delete_items
-    , unsafe_update_items
-    , unsafe_delete_items
-    , videogames
-    , designers
-    , students
-    , students_info
-    , students_view
-    , students_info_view
-    , country
-    , capital
-    , first
-    , second
-    , first_1
-    , second_1
-    , fee
-    , baz
-    , janedoe
-    , johnsmith
+    , v1
+    , v2
 TO postgrest_test_anonymous;
 
+REVOKE ALL PRIVILEGES ON TABLE
```

Comment:
I like this change but seems there are some tables missing.

On the explicit GRANT list above there were `197` tables.

And on the test schema we have

```
$ postgrest-with-postgresql-14 psql

select count(*) from information_schema.tables where table_schema = 'test';
 count
-------
   220
```

So I guess there are 20 tables missing from the REVOKE list?

Reply:
`insertonly` for example is missing.

```
select count(*)
from information_schema.tables
where table_schema = 'test' and table_name not in (
     'items'
    ,'items2'
    ,'items3'
    ,'articleStars'
    ,'articles'
    ,'arrays'
    ,'auto_incrementing_pk'
    ,'clients'
    ,'comments'
    ,'complex_items'
    ,'compound_pk'
    ,'compound_pk_view'
    ,'deferrable_unique_constraint'
    ,'empty_table'
    ,'fav_numbers'
    ,'has_count_column'
    ,'has_fk'
    ,'insertable_view_with_join'
    ,'json_table'
    ,'materialized_view'
    ,'menagerie'
    ,'no_pk'
    ,'nullable_integer'
    ,'projects'
    ,'projects_view'
    ,'projects_view_alt'
    ,'test_null_pk_competitors_sponsors'
    ,'simple_pk'
    ,'simple_pk2'
    ,'tasks'
    ,'filtered_tasks'
    ,'tsearch'
    ,'users'
    ,'users_projects'
    ,'users_tasks'
    ,'files'
    ,'touched_files'
    ,'Escap3e;'
    ,'ghostBusters'
    ,'withUnique'
    ,'clashing_column'
    ,'موارد'
    ,'addresses'
    ,'orders'
    ,'consumers_view'
    ,'consumers_view_view'
    ,'consumers_extra_view'
    ,'orders_view'
    ,'images'
    ,'images_base64'
    ,'w_or_wo_comma_names'
    ,'items_with_different_col_types'
    ,'entities'
    ,'child_entities'
    ,'grandchild_entities'
    ,'ranges'
    ,'being'
    ,'descendant'
    ,'being_part'
    ,'part'
    ,'leak'
    ,'perf_articles'
    ,'employees'
    ,'tiobe_pls'
    ,'single_unique'
    ,'compound_unique'
    ,'only_pk'
    ,'family_tree'
    ,'managers'
    ,'organizations'
    ,'authors'
    ,'publishers'
    ,'books'
    ,'forties_books'
    ,'fifties_books'
    ,'sixties_books'
    ,'person'
    ,'message'
    ,'person_detail'
    ,'space'
    ,'zone'
    ,'projects_dump'
    ,'UnitTest'
    ,'json_arr'
    ,'jsonb_test'
    ,'authors_books_number'
    ,'authors_have_book_in_decade'
    ,'authors_have_book_in_decade2'
    ,'forties_and_fifties_books'
    ,'odd_years_publications'
    ,'foos'
    ,'bars'
    ,'materialized_projects'
    ,'contract'
    ,'player_view'
    ,'contract_view'
    ,'ltree_sample'
    ,'isn_sample'
    ,'projects_count_grouped_by'
    ,'Server Today'
    ,'pgrst_reserved_chars'
    ,'authors_w_entities'
    ,'openapi_types'
    ,'openapi_defaults'
    ,'getallprojects_view'
    ,'get_projects_above_view'
    ,'web_content'
    ,'pages'
    ,'referrals'
    ,'big_projects'
    ,'sites'
    ,'jobs'
    ,'main_jobs'
    ,'whatev_projects'
    ,'whatev_sites'
    ,'whatev_jobs'
    ,'agents'
    ,'departments'
    ,'schedules'
    ,'activities'
    ,'unit_workdays'
    ,'unit_workdays_fst_shift'
    ,'stuff'
    ,'loc_test'
    ,'v1.parents'
    ,'v2.parents'
    ,'v2.another_table'
    ,'v1.children'
    ,'v2.children'
    ,'screens'
    ,'labels'
    ,'label_screen'
    ,'actors'
    ,'films'
    ,'personnages'
    ,'end_1'
    ,'end_2'
    ,'schauspieler'
    ,'filme'
    ,'rollen'
    ,'products'
    ,'suppliers'
    ,'products_suppliers'
    ,'trade_unions'
    ,'suppliers_trade_unions'
    ,'client'
    ,'clientinfo'
    ,'contact'
    ,'chores'
    ,'limited_update_items'
    ,'limited_update_items_cpk'
    ,'limited_update_items_no_pk'
    ,'limited_update_items_view'
    ,'limited_update_items_wnonuniq_view'
    ,'limited_delete_items'
    ,'limited_delete_items_cpk'
    ,'limited_delete_items_no_pk'
    ,'limited_delete_items_view'
    ,'plate'
    ,'well'
    ,'limited_delete_items_wnonuniq_view'
    ,'limited_delete_items_cpk_view'
    ,'limited_update_items_cpk_view'
    ,'xmltest'
    ,'oid_test'
    ,'job'
    ,'series'
    ,'adaptation_notifications'
    ,'series_popularity'
    ,'test'
    ,'view_test'
    ,'shops'
    ,'shop_bles'
    ,'do$llar$s'
    ,'safe_update_items'
    ,'safe_delete_items'
    ,'unsafe_update_items'
    ,'unsafe_delete_items'
    ,'videogames'
    ,'designers'
    ,'students'
    ,'students_info'
    ,'students_view'
    ,'students_info_view'
    ,'country'
    ,'capital'
    ,'first'
    ,'second'
    ,'first_1'
    ,'second_1'
    ,'fee'
    ,'baz'
    ,'janedoe'
    ,'johnsmith'
);

 count
-------
    34

```

Removed from the list:
```
    , public.public_consumers
    , public.public_orders
    , "SPECIAL ""@/\#~_-".languages
    , "SPECIAL ""@/\#~_-".names
```

Reply:
These are missing

```
 private_table
 sponsors
 competitors
 bar
 Foo
 projects_auto_updatable_view_without_pk
 projects_auto_updatable_view_with_pk
 projects_view_without_triggers
 projects_view_with_all_triggers_with_pk
 projects_view_with_all_triggers_without_pk
 projects_view_with_insert_trigger
 projects_view_with_update_trigger
 projects_view_with_delete_trigger
 car_models
 car_models_2021
 car_models_default
 car_brands
 car_model_sales_202101
 car_model_sales
 car_model_sales_default
 car_racers
 car_dealers
 car_dealers_springfield
 car_dealers_default
 car_models_car_dealers
 xml
 no_pk_view
 plate_plan_step
 insertonly
 car_models_car_dealers_10to20
 car_models_car_dealers_default
```